### ISSUE TITLE
[Site-Support-Request] Plays.tv

### ISSUE THREAD

**Issue by Delivator** [rg3/youtube-dl#8458]
Main Site:
`http://plays.tv`

Examples: 

Site:
`http://plays.tv/video/569aa4145e8de99951` or
`http://plays.tv/video/569aa4145e8de99951/knife-killing-knife-rekt-bo3-bops3-cod`

Embedded:
`http://plays.tv/embed/569aa4145e8de99951`
Or with likes and comments:
`http://plays.tv/embeds/569aa4145e8de99951`

Raw:
`http://m1playscdntv-a.akamaihd.net/video/AsZOJ2F0N-U/processed/360.mp4` 360
`http://m1playscdntv-a.akamaihd.net/video/AsZOJ2F0N-U/processed/480.mp4` 480
`http://m1playscdntv-a.akamaihd.net/video/AsZOJ2F0N-U/processed/720.mp4` 720
`http://m1playscdntv-a.akamaihd.net/video/AsZOJ2F0N-U/processed/720H.mp4` 720H
`http://m1playscdntv-a.akamaihd.net/video/AsZOJ2F0N-U/processed/1080H.mp4` 1080H

Thumbnails:
`http://m1playscdntv-a.akamaihd.net/video/AsZOJ2F0N-U/processed/360.jpg` 360
`http://m1playscdntv-a.akamaihd.net/video/AsZOJ2F0N-U/processed/480.jpg` 480
`http://m1playscdntv-a.akamaihd.net/video/AsZOJ2F0N-U/processed/720.jpg` 720

You already can donwload videos, but only in one quallity (480p)

**Comment by Delivator** [comment]
I've updated some better examples!

**Comment by remitamine** [comment]
this site will be supported in the next version but i will keep this issue open untill we fix dash format sorting.

**Comment by remitamine** [comment]
format sorting will be fixed in the next version.
```
python __main__.py -F http://plays.tv/video/569aa4145e8de99951
[PlaysTV] 569aa4145e8de99951: Downloading webpage
[PlaysTV] 569aa4145e8de99951: Downloading MPD manifest
[info] Available formats for 569aa4145e8de99951:
format code       extension  resolution note
DASH-AUDIO-192k   m4a        audio only DASH audio  196k , mp4a.40.2 (48000Hz)
DASH-VIDEO-360    mp4        640x360    DASH video  659k , avc1.64001e, 30fps, video only
DASH-VIDEO-480    mp4        852x480    DASH video  869k , avc1.64001f, 30fps, video only
DASH-VIDEO-720    mp4        1280x720   DASH video 2211k , avc1.64001f, 30fps, video only
DASH-VIDEO-720H   mp4        1280x720   DASH video 4151k , avc1.640020, 31fps, video only
DASH-VIDEO-1080H  mp4        1920x1080  DASH video 5938k , avc1.64002a, 31fps, video only
http-360          mp4        360p       
http-480          mp4        480p       
http-720          mp4        720p       
http-720h         mp4        720p       
http-1080h        mp4        1080p      (best)
```

**Comment by Delivator** [comment]
Awesome thank you!
